README.md

```diff
@@ -35,14 +35,16 @@ Getting started
 5. Test: Visit http://localhost:8080/index.html
 6. (Optionally) Have your own [tanner](https://github.com/mushorg/tanner) service running.
 
+[Note : Cloner clones the whole website, to restrict to a desired depth of cloning add `--max-depth` parameter]
+
 #### Docker build instructions
 1. Change current directory to `snare` project directory
 2. `docker-compose build`
 3. `docker-compose up`
-
+4. Snare will start on 0.0.0.0 with port 80.
 More information about running `docker-compose` can be found [here.](https://docs.docker.com/compose/gettingstarted/)
 
-[Note : Cloner clones the whole website, to restrict to a desired depth of cloning add `--max-depth` parameter]
+In case of an error while running `docker-compose up`, check the availability of port 80, if it is occupied then reffer to docker documentations to change the default port.
```

Comment:
`reffer` -> `refer`
`documentations` -> `documentation`
`docker` -> `Docker`


I know that all that changes might be exhausting, but REAME is the "face" of the project, so try to understand :)

Reply:
Yes definitely, i am always ready to make corrections, also i am sorry for continuously making these mistake :sweat_smile:  
thank you for reviewing my PR :smile: